src/test/fixtures/mocks/response.js

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This code isn't new, I moved it from `src/test/unit/shorten-url.test.js` (see below), and added the method `async text`.